contracts/verification/LivepeerVerifier.sol

```diff
@@ -55,6 +55,14 @@ contract LivepeerVerifier is Manager, IVerifier {
         verificationCodeHash = _verificationCodeHash;
     }
 
+    function addSolver(address _solver) external onlyControllerOwner {
+        // Must not already be a solver
+        require(!isSolver[_solver]);
```

Comment:
We did `require(!isSolver[_solvers[i]] && _solvers[i] != address(0));` earlier, should we do `address(0)` check here too?